Tests/varLib/instancer_test.py

```diff
@@ -279,57 +299,81 @@ def test_scaleVarDataDeltas(self):
 
         instancer._scaleVarDataDeltas(varData, regionScalars)
 
-        assert varData.Item == [[50, 0], [-50, 0]]
+        assert varData.Item == [[50], [-50]]
+
+    def test_popVarDataDeltas(self):
+        varData = builder.buildVarData([1, 0], [[34, 68], [-100, -200]], optimize=False)
+        assert instancer._popVarDataDeltas(varData, 1) == [34, -100]
+        assert varData.VarRegionCount == 1
+
+        varData = builder.buildVarData([1, 0], [[34, 68], [-100, -200]], optimize=False)
+        assert instancer._popVarDataDeltas(varData, 0) == [68, -200]
+        assert varData.VarRegionCount == 1
+
+        varData = builder.buildVarData([1, 0], [[34, 68], [-100, -200]], optimize=False)
+        assert instancer._popVarDataDeltas(varData, 2) == [0, 0]  # missing
+        assert varData.VarRegionCount == 2
```

Comment:
Can you add another test case: instancer._popVarDataDeltas(varData, None). I don't think we would want [0, 0] as the output here?